sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/src/AzureMonitorLogExporter.cs

```diff
@@ -26,11 +27,17 @@ internal AzureMonitorLogExporter(ITransmitter transmitter)
             _transmitter = transmitter;
             _instrumentationKey = transmitter.InstrumentationKey;
             _resourceParser = new ResourceParser();
+
+            // TODO: Add check if offline storage is enabled by user via options
+            // If disabled do not initialize AzureMonitorPersistentStorage
+            _persistentStorage = new AzureMonitorPersistentStorage(transmitter);
```

Comment:
Should we just add check here? if (_transmitter.storage != null)

Reply:
This is a nice idea and we will have all implementations within `AzureMonitorTransmitter`. Will send a follow up PR for this one.